### ISSUE TITLE
NVDA 2019.1 - Odd interaction issues with MS-Word 2016 Fillable Form document

### ISSUE THREAD

**Issue by britechguy** [nvaccess/nvda#9411]
### Steps to reproduce:
1. Open attached BusNote_Document.docx file
2. See notes under Actual Behavior

### Actual behavior:
1. File opens, dead silence ensues, although focus in word itself is in the first form field, Teacher/Grade.
2. Switch focus to any other open window, then back to the document, NVDA now announces that it is working with BusNote_Document.docx, and says "multi-line, selected."
3. Sometimes, and I cannot replicate this consistently, if one begins tabbing through the form fields and checkboxes, NVDA consistently says, "Blank."  If I hit F1 when this is occurring, to make help text display, afterward it begins announcing the labels for the text boxes and checkboxes upon entry to each.
4. Checkboxes are not announced with their checked/unchecked state noted.  For example, the first checkbox in the form is, "Ride Bus number," and when it lands on it NVDA announces, "Ride Bus Number Checkbox, checkbox"

### Expected behavior:
When I land in any form field or checkbox, I always expect its label/name to be announced.  When the object is a checkbox, I expect that the name will be announced followed immediately by, "Checkbox checked/unchecked," depending on the state of the checkbox itself.  I did just determine that if I check a checkbox, navigate away from it, then come back to it the status, "Checkbox checked," is announced when I do come back to it.   It also strikes me as overkill to announce something that is a checkbox as a checkbox, followed by the phrase "checkbox checked."  A simple Checked or unchecked would do since you've already had the object identified as a checkbox.
[BusNote_Document.docx](https://github.com/nvaccess/nvda/files/3009990/BusNote_Document.docx)

#### NVDA installed/portable/running from source:
NVDA Installed

#### NVDA version:
2019.1

#### Windows version:
Windows 10 Home, Version 1809, Build 17763.379, 64-bit

#### Name and version of other software in use when reproducing the issue:
Microsoft Word 2016

#### Other information about your system:
HP Laptop, AMD A12-9700P processor, Radeon R7 Graphics, 12GB RAM

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by britechguy** [comment]
I just determined that the "checkbox checkbox" announcement is my own fault.  I had forgotten that the help text on the checkboxes all include the word "checkbox" as part of it, so what's being announced as the "checkbox checkbox" is my help text label word plus NVDA identifying the actual type of the object.

The other issues, though, remain as described.

**Comment by britechguy** [comment]
Revised Bus Note document to get rid of the "checkbox checkbox" issue caused by my own help text.  The other issues remain.
[BusNote_V2.docx](https://github.com/nvaccess/nvda/files/3013655/BusNote_V2.docx)

If anyone wants a copy of the .dotx file, which is used to generate "fresh copies" of the Bus Note as they need to be filled out, contact me.  I can't attach a .dotx file here.

**Comment by Adriani90** [comment]
Status with NVDA 2024.2 Beta, UIA disabled, MS Office 365:
Issues 1 through 4 are not reproducible
1. NVDA reports directly when the focus is in the document after opening the file, but it doesn't report the label of the focused edit field. This is because in MS Word we don't have any accessible name calculation for edit fields, this needs to be fixed by Microsoft. However, if the edit field of teacher was filled with some content, NVDA would report that content when opening the file.
2. Obsolete, everything works right there.
3. Tabbing through the form doesn't say "blank" for me, it adds some tab characters to the edit fields though. This is something that is implemented by design through Microsoft and NVDA cannot do anything about it.
4. Not reproducible anymore. State of check boxes is reported correctly when tabbing on them, but it is not reported when changing the state with spacebar. This last part of the issue is highly related to #15124.

@britechguy how is your experience with NVDA 2024.2 Beta in this document? Note that you have to disable UIA in NVDA advanced settings, otherwise this document will be totally inaccessible.
